### PR TITLE
Improved protobuf

### PR DESCRIPTION
**Pull request by slott56**
Warning: massive change

1. Improve some protobuf handling in general. This introduces better name handling so that a single name can have a be combination of type and value references. Names start with type associations. Some names also get values.

2. Fix the conversion of test cases from protobuf text to Gherkin.

### CODE REVIEW

**Comment by slott56** [comment]
Closes #17 and #3.

File: src/celpy/c7nlib.py [hunk 1]
```diff
@@ -52,8 +52,8 @@ def process(self,
             for resource in resources:
                 with C7NContext(filter=the_filter):
                     cel_activation = {
-                        "resource": celpy.json_to_cel(resource),
-                        "now": celpy.celtypes.TimestampType(now),
+                        "Resource": celpy.json_to_cel(resource),
```
Comment (kylejohnson514) [inline src/celpy/c7nlib.py:55]: Need to update these `Resource`, `Now`, and `Event` fields throughout the codebase to be `resource`, `now`, and `event`

**Review by kylejohnson514 — CHANGES_REQUESTED** [review]
Capitalization of the `Resource`, `Now`, and `Event` fields needs to be pulled in from master, but once those are resolved, the rest of this PR LGTM!

**Review by kylejohnson514 — APPROVED** [review]
LGTM, I tested this locally with some CEL expressions in Custodian as well

**Pull request merged**